### ISSUE TITLE
Helm plugin install broken

### ISSUE THREAD

**Issue by karlkfi** [databus23/helm-diff#42]
```
$ helm plugin install https://github.com/databus23/helm-diff
...
Downloading https://github.com/databus23/helm-diff/releases/download/v2.8.2%2B2/helm-diff-linux.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   609    0   609    0     0   2845      0 --:--:-- --:--:-- --:--:--  2845
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  6 4749k    6  288k    0     0   216k      0  0:00:21  0:00:01  0:00:20  305k 34 4749k   34 1648k    0     0   697k      0  0:00:06  0:00:02  0:00:04  833k 66 4749k   66 3144k    0     0   948k      0  0:00:05  0:00:03  0:00:02 1073k100 4749k  100 4749k    0     0  1155k      0  0:00:04  0:00:04 --:--:-- 1275k
Preparing to install into /root/.helm/plugins/helm-diff
cp: can't create '/root/.helm/plugins/helm-diff/diff': File exists
Failed to install helm-diff
\tFor support, go to https://github.com/databus23/helm-diff.
Error: plugin install hook for "diff" exited with error
```

This happens when installing on `alpine:3.7` in my Dockerfile with the step:

```
ENV HELM_VERSION v2.8.2
RUN curl -sfL https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz \
  | tar -xz -C /bin/ --strip-components=1 linux-amd64/helm
RUN helm init --client-only \
  && helm plugin install https://github.com/databus23/helm-diff
```

**Comment by databus23** [comment]
Urks, this caused by my restructuring of the code apparently. Thanks for reporting this.

**Comment by databus23** [comment]
Fixed by 380325c4249b20fcbbfedb55865729dfbb18d363
